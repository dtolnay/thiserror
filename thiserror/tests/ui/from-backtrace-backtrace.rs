// https://github.com/dtolnay/thiserror/issues/163

use std::backtrace::Backtrace;
use thiserror::Error;

#[derive(Error, Debug)]
#[error("...")]
pub struct Error(
    #[from]
    #[backtrace]
    std::io::Error,
    Backtrace,
);

fn main() {}
