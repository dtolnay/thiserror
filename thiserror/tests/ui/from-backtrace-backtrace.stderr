error: deriving From requires no fields other than source and backtrace
 --> tests/ui/from-backtrace-backtrace.rs:9:5
  |
9 |     #[from]
  |     ^^^^^^^
